etc/opensc.conf.oseid

```diff
@@ -0,0 +1,14 @@
+# This is configuration file used for running the OsEID emulator in CI
+
+app default {
+        card_atr 3b:f5:00:00:02:10:80:4f:73:45:49:44 {
+                atrmask = "ff:ff:00:ff:ff:ff:ff:ff:ff:ff:ff:ff";
+                driver = "myeid";
+        }
+        # new ATR for OsEID T0, T1 protocol
+        card_atr 3b:f5:00:00:02:80:01:4f:73:45:49:44:00 {
+                #force_protocol = t0;
+                atrmask = "ff:ff:00:ff:ff:ff:ff:ff:ff:ff:ff:ff:00"
+                driver = "myeid";
+        }
+}
```

Comment:
Does the standard configuration not work?

Reply:
This was part of the instructions I got from @popovec. It does not work without that:

https://travis-ci.org/Jakuje/OpenSC/builds/542173526

It would be better to have this part in the driver, rather than in separate configuration files. The MyEID card detection checks for this string in the ATR historical bytes so it fails for the OsEID historical bytest. But it should be pretty trivial to modify the myeid driver, then it works:

https://travis-ci.org/Jakuje/OpenSC/builds/542176862

If you agree, I will add the commits from there to this PR and squash them.

Reply:
Somebody from the Aventra should comment on this.

Reply:
For the record, there was a comment in [1] from @hhonkanen approving the changes with a suggestion to log what card was matched and set the card name to the OsEID. This is what I did in 
https://github.com/Jakuje/OpenSC/commit/79f63827bc74cf5d3d064a1e585d62200c28e213
I will add the changes also to this PR and then we are ready for merge.

[1] https://github.com/Jakuje/OpenSC/commit/94adc80037bad65b3990c11c6289c53c619b43f7#commitcomment-33906504